src/Routes/SeasonsController.ts

```diff
@@ -2,8 +2,37 @@ import * as express from "express";
 import { IDataProvider } from "../DataProviders/IDataProvider";
 import { BaseController } from "./BaseController";
 
-/*
- * Seasons controller
+/**
+ * @swagger
+ *
+ * definitions:
+ *   Seasons:
+ *     type: object
```

Comment:
array

Reply:
This is the object definition, I use the reference later on in the schema